Fix user password update

vitrez opened the pull request:
##### SUMMARY
<!--- Describe the change below, including rationale and design decisions -->
Add `ON CLUSTER` clause to all update user functions
<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
Fixes #104 

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Bugfix Pull Request

##### COMPONENT NAME
<!--- Write the short name of the module, plugin, task or feature below -->
clickhouse_user

Andersson007 commented:
@vitrez hello, welcome and thanks for fixing the bug!
Could you please also add a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment)?

Andersson007 commented:
@vitrez please remove what's been added in the last commit and add the [fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment): the changelog is automatically generated based on the fragments when releasing
Thanks

Andersson007 commented:
@aleksvagachev please take a look

vitrez commented:
@Andersson007 hello, thanks for your prompt response. Sorry, this is my first commit to ansible collection repo. I hope I did everything right now. I added file `105-clickhouse_user.yaml` and removed my manual changes to CHANGELOG.

aleksvagachev commented:
@vitrez Thanks for the contribution!

Andersson007 commented:
@aleksvagachev thanks for reviewing!
@vitrez thanks a lot for the contribution! We welcome new contributors here ...and it's easy to become a maintainer!
So I hope we'll see more contributions from you!
Take a look at open issues:
- Just comments/opinion sharing are much appreciated
- Feel free to open new issues
- If you want to fix anything yourself, there's also a [dev guide](https://docs.ansible.com/ansible/devel/community/create_pr_quick_start.html) that can be helpful
- If you need any mentoring? We are on Matrix (andersson007_ and aleks_vagachev) and [Forum](https://forum.ansible.com/g/ClickHouseTeam), see the [communication guide](https://docs.ansible.com/ansible/devel/community/communication.html).

vitrez commented:
@Andersson007 @aleksvagachev thanks for accepting my PR. Can we expect version 0.8.2 to appear in ansible galaxy with the changes made?

Andersson007 commented:
@vitrez sure, done, here's a copy of the announcement:
The [community.clickhouse](https://galaxy.ansible.com/ui/repo/published/community/clickhouse) collection version [0.8.2](https://github.com/ansible-collections/community.clickhouse/releases/tag/0.8.2) has been released!
